plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgreSqlConnectorTest.java

```diff
@@ -103,6 +103,7 @@ protected QueryRunner createQueryRunner()
         postgreSqlServer = closeAfterClass(new TestingPostgreSqlServer());
         return PostgreSqlQueryRunner.builder(postgreSqlServer)
                 .setInitialTables(REQUIRED_TPCH_TABLES)
+                .withProtocolSpooling("json")
```

Comment:
would that make sense to run both versions?

Reply:
Yeah, maybe so